rotkehlchen/externalapis/coingecko.py

```diff
@@ -294,6 +296,12 @@ def simple_price(self, from_asset: Asset, to_asset: Asset) -> Price:
             )
             return Price(ZERO)
 
+    def rate_limited_in_last(self, seconds: int = COINGECKO_RATE_LIMIT_WAIT_TIME) -> bool:
```

Comment:
I've added this method for being able to standardise the calls across the current and historical price oracles.

Reply:
I've seen PR #2183 adds and relies on `can_query_history`. I'll need to take it into account